module/x/gravity/migrations/v2/keys.go

```diff
@@ -0,0 +1,26 @@
+package v2
+
+import "strings"
+
+// Store prefixes
+const (
+	StoreKey = "gravity"
+
+	DenomToERC20Key = "DenomToERC20Key"
+	ERC20ToDenomKey = "ERC20ToDenomKey"
+
+	EthAddressByValidatorKey = "EthAddressValidatorKey"
+	ValidatorByEthAddressKey = "ValidatorByEthAddressKey"
+
+	BatchConfirmKey    = "BatchConfirmKey"
+	OutgoingTXPoolKey  = "OutgoingTXPoolKey"
+	OutgoingTXBatchKey = "OutgoingTXBatchKey"
+)
+
+func ConvertByteArrToString(value []byte) string {
```

Comment:
do we need this function here? I'm not really sure why we don't just use the native cast to string? Maybe this is a go nuance I'm missing.

Reply:
I was asking myself the same question the other day, because `ConvertByteArrToString` and simply casting to string don't produce the same result (it looks like `ConvertByteArrToString` adds some extra bytes somehow).
I talked with @alexanderbez about this and he also suggested we just use `string(...)`
I can remove it from the one that's using it (`OutgoingTXPoolKey`).

Reply:
Yes, ACK to direct string casting! I think `ConvertByteArrToString` adds extra bytes as it converts each _byte_ to a string.

Reply:
This method produces bad results in some cases. Some of the keys that exist in the store that were created by using ConvertByteArrToString could be malformed as well. For this reason we removed this method usage from key creation in all cases but one. In this one case (GetPastEthSignatureCheckpointKey()) we couldn't reconstruct the valid key so the only other option was to delete all those keys from the store, which is not acceptable. We also noted that this method should not be used anywhere else.

Reply:
@dusan-ethernal so should I leave it as is and you take care of removing it in another PR?

Reply:
We saw that you already reverted to not using this method as Christian suggested. This looks good to us, we will proceed from there.